substrate/mmap

```diff
@@ -4,14 +4,18 @@
 
 #include <cstdint>
 #include <substrate/internal/defs>
-#ifndef _WINDOWS
-#	include <unistd.h>
-#	include <sys/mman.h>
-#else
-#	define NOMINMAX
-#	define WIN32_LEAN_AND_MEAN
+#ifdef _WIN32
```

Comment:
In substrate/internal/defs there's the last remnant of the `_WINDOWS` macro.

Reply:
Think we'll do a clean-up PR fixing all `_WINDOWS` remaining macro use by replacement with `_WIN32`. That way we can fix it right